src/Common/tests/Scripts/Tools/ParallelTestExecution.ps1

```diff
@@ -14,7 +14,7 @@
 
 function BuildAndTestBinary
 {
-    $output = (msbuild /t:rebuild,test /p:Outerloop=true)
+    $output = (msbuild /t:rebuild,test /p:OuterLoop=true)
```

Comment:
I didn't even know this script existed. Should we delete it?

Reply:
Hmm yeah but not as part of this PR :)